README.md

```diff
@@ -49,6 +49,12 @@ With Standard Containers we can put an end to that embarrassment, by making INDU
 The specification and code is licensed under the Apache 2.0 license found in 
 the `LICENSE` file of this repository.  
 
+## Markdown style
+
+To keep consistency throughout the markdown files in the open container spec all files should be formatted one sentence per-line.
+This fixes two things: makes diffing easier with the git tooling and resolves fights about line wrapping length.
+So, for example this paragraph, in raw text, will span three-lines.
```

Comment:
And I'd probably shuffle the last sentence around to get “For example, this paragraph will span three lines in the Markdown source.”